eZ/Publish/Core/Repository/SiteAccessAware/SectionService.php

```diff
@@ -75,6 +76,11 @@ public function assignSection(ContentInfo $contentInfo, Section $section)
         return $this->service->assignSection($contentInfo, $section);
     }
 
+    public function assignSectionToSubtree(Location $location, Section $section): void
```

Comment:
please add a docblock

Reply:
This class is a decorator for Repo, so doc has been skipped by design, alternative compromise to avoid matianing doc across many classes when it's source is the interface is a `/** {@inheritDoc} */` like is the case in `Core/SignalSlot`.